src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -6,64 +6,196 @@
 #
 import random as _random
 import turicreate as _tc
+import turicreate.toolkits._internal_utils as _tkutl
 from turicreate import extensions as _extensions
 from turicreate.toolkits._model import CustomModel as _CustomModel
+from turicreate.toolkits._model import PythonProxy as _PythonProxy
+from turicreate.toolkits.object_detector.object_detector import ObjectDetector as _ObjectDetector
 
 def create(dataset,
            target,
+           augmentation_mode="auto",
+           flip_horizontal=True,
+           flip_vertical=True,
            backgrounds=None,
-           feature=None,
            batch_size=0,
            max_iterations=0,
            seed=None,
-           verbose=True):
-    model = _extensions.one_shot_object_detector()
+           verbose=True,
+           **kwargs):
+    """
+    Create a :class:`ObjectDetector` model.
+
+    Parameters
+    ----------
+
+    dataset : SFrame | tc.Image
+        An SFrame that contains all the starter images along with their
+        corresponding labels.
+        If the dataset is a single tc.Image, this parameter is just that image.
+        Every starter image should entirely be just the starter image without
+        any padding.
+        RGB and RGBA images allowed.
+
+    target : string
+        The target column name in the SFrame that contains all the labels. 
+        If the dataset is a single tc.Image, this parameter is just a label for
+        that image.
+
+    augmentation_mode : string optional
+        An augmentation mode is a shortcut to setting the 'flip_horizontal',
```

Comment:
So is this parameter ignored if you pass in values for `kwargs`? Do `kwargs` take precedence over this parameter?

Reply:
Anything passed into kwargs takes precedence. However, it's slightly different from completely ignoring the parameter. Consider a scenario where the user passes in `augmentation_mode="logo"`, but also passes in `flip_horizontal=True`. All the pre-set values that come from `augmentation_mode="logo"` still stand, except for `flip_horizontal`.